examples/nidaqmx_analog_input/measurement.py

```diff
@@ -28,6 +28,11 @@
 )
 service_options = ServiceOptions()
 
+# If another measurement is using the session, the reserve function will wait
+# for it to complete. Specify a reservation timeout to aid in debugging missed
+# unreserve calls. Long measurements may require a longer timeout.
+RESERVATION_TIMEOUT_IN_SECONDS = 60.0
+
```

Comment:
consider:
```suggestion
RESERVATION_TIMEOUT_IN_SECONDS = 60.0
"""
If another measurement is using the session, the reserve function will wait
for it to complete. Specify a reservation timeout to aid in debugging missed
unreserve calls. Long measurements may require a longer timeout.
"""

```

this tells Sphinx and ide's to treat this as a doc-string for this constant.

Reply:
Done. I also did this for a couple other constants.